### PR TITLE
Update README with Mac install guide, various other adjustments to content + formatting

### PR DESCRIPTION
**Pull request by ethangreen-dev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,7 +117,7 @@ name = "nativefs"
 
 ### Patch variants
 
-This file contains two patch definitions - a pattern patch, which (currently) changes a single line at a position offset to some pattern match, and a copy patch, which reads one or more input lua files and either appends or prepends them onto the target. The former is used when you need to surgically embed code at specific locations in the target (very useful for modloader init routines), and the latter is designed for use when you need to bulk inject position-independent code into the game.
+This file contains three patch definitions - a pattern patch, which (currently) changes a single line at a position offset to some pattern match, and a copy patch, which reads one or more input lua files and either appends or prepends them onto the target. The former is used when you need to surgically embed code at specific locations in the target (very useful for modloader init routines), and the latter is designed for use when you need to bulk inject position-independent code into the game.
```
Comment (english5040) [inline README.md:120]: There are 4 types of patch now; also only 2 are described

Comment (english5040) [reply README.md:120]: maybe just use bullet points?

**Review by english5040 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ethangreen-dev) [reply README.md:120]: yeah, I was avoiding updating this paragraph. will do this tomorrow, thanks 👍

File: README.md [hunk 2]
```diff
@@ -67,23 +114,26 @@ before = "main.lua"
...
+- Use `module` patches to inject a lua module into the game's runtime. Note that this currently only supports single file modules, but this should be changing soon.
 
 ### Patch files
 
-Patch files are loaded from mod directories inside of `%AppData%/Balatro/Mods`. Lovely will load any patch files present within `Mods/ModName/lovely/` or load a single patch from `%AppData/Balatro/Mods/ModName/lovely.toml`. If multiple patches are loaded they will be injected into the game in the order in which they are found.
+Patch files are loaded from mod directories inside of the mod folder (`MOD_DIR`). Lovely will load any patch files present within `MOD_DIR/ModName/lovely/` or load a single patch from `MOD_DIR/ModName/lovely.toml`. If multiple patches are loaded they will be injected into the game in the order in which they are found.
 
-Paths defined within the patch are rooted by the mod's directory. For example, `core/deck.lua` is resolved to `%AppData%/Balatro/Steamodded/core/deck.lua`.
+Paths defined within the patch are rooted by the mod's directory. For example, `core/deck.lua` is resolved to `MOD_DIR/Steamodded/core/deck.lua`.
```
Comment (english5040) [inline README.md:128]:
```suggestion
Paths defined within the patch are rooted by the mod's directory. For example, `core/deck.lua` resolves to `MOD_DIR/ModName/core/deck.lua`.
```

File: README.md [hunk 3]
```diff
@@ -67,23 +114,26 @@ before = "main.lua"
 name = "nativefs"
 ```
 
-### Patch variants
+### TL;DR - Patch variants
 
-This file contains two patch definitions - a pattern patch, which (currently) changes a single line at a position offset to some pattern match, and a copy patch, which reads one or more input lua files and either appends or prepends them onto the target. The former is used when you need to surgically embed code at specific locations in the target (very useful for modloader init routines), and the latter is designed for use when you need to bulk inject position-independent code into the game.
+- Use `pattern` patches are used to surgically embed code at specific locations within the target. Supports the `*` and `+` wildcards.
```
Comment (english5040) [inline README.md:119]:
```suggestion
- Use `pattern` patches to surgically embed code at specific locations within the target. Supports the `*` and `+` wildcards.
```

Comment (english5040) [reply README.md:119]: Which wildcards are supported again? A comment said `*` and `?`

**Review by english5040 — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (ethangreen-dev) [reply README.md]: Good catch, it's supposed to be `*` and `?`.

**Review by english5040 — APPROVED** [review]

**Pull request merged**